src/main/resources/patch_4.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE island_blocks ADD COLUMN extra_amount INTEGER PRECISION DEFAULT 0.00;
+ALTER TABLE island_spawners ADD COLUMN extra_amount INTEGER PRECISION DEFAULT 0.00;
```

Comment:
I don't think that these SQL statements work with MySQL. Ran a checker I found online, and it shows errors.